docs/hadoop.txt

```diff
@@ -65,8 +65,8 @@ gremlin> g.V().count()
 # Hadoop Graph Configuration
 #
 gremlin.graph=org.apache.tinkerpop.gremlin.hadoop.structure.HadoopGraph
-gremlin.hadoop.graphInputFormat=org.apache.tinkerpop.gremlin.hadoop.structure.io.gryo.GryoInputFormat
```

Comment:
Note that this file has been renamed to `hadoop.adoc` recently; please port your changes to the new file.